doc/script_commands.txt

```diff
@@ -2473,6 +2473,7 @@ arrays:
 ---------------------------------------
 
 *readparam(<parameter number>{, "<player name>"})
+*readparam(<parameter number>{, <account id>})
```

Comment:
we have to standardize this one

in `*getcharip` and [`*setgroupid`](https://github.com/HerculesWS/Hercules/blob/stable/doc/script_commands.txt#L3416), it separated with `|`
in [`*message`](https://github.com/HerculesWS/Hercules/blob/stable/doc/script_commands.txt#L4387-L4388) however, it separated with new line

Reply:
actually a lot of buildins use this syntax to show alternate form:
- getmapinfo
- getareausers
- getgdskilllv
- getskilllv
- checkweight
- message
- recovery <= but this one should be changed to `recovery({<account id})
- montransform
- getitem
- getitem2
- getitembound
- getitembound2
- getnameditem
- rentitem
- makeitem
- makeitem2